apps/designer/app/designer/features/sidebar-left/panels/asset-manager/utils.ts

```diff
@@ -0,0 +1,22 @@
+export const getHumanReadableFileSize = (size: number) => {
```

Comment:
I bet there are battle tested libs for this, is there a small and a really good one?

Reply:
or maybe I am overestimating the complexity there?

Reply:
e.g. this one https://github.com/sindresorhus/pretty-bytes/

Reply:
changed to it, its en esm so I added it to the dependencies to bundle